docs/articles/index.md

```diff
@@ -0,0 +1,136 @@
+# Index
+
+[A](#a) · [B](#b) · [C](#c) · [D](#d) · [E](#e) · [F](#f) · [G](#g) · [H](#h) · [I](#i) · [J](#j) · [K](#k) · [L](#l) · [M](#m) · [N](#n) · [O](#o) · [P](#p) · [Q](#q) · [R](#r) · [S](#s) · [T](#t) · [U](#u) · [V](#v) · [W](#w) · [X](#x) · [Y](#y) · [Z](#z)
+
+---
+
+## A
+[Access Control](access-control.md)  
+[ACLs](access-control.md)  
+[API](api.md)  
+[Application Groups](application-groups.md)  
```

Comment:
We don't really support groups for pods (other than pods having a pathId). I think this was targeting groups as
* scale a group
* create/update a group
* dependencies

I'm OK with renaming but I imagine it would feel like _not supported for pods_ instead of _feature is likely to be deprecated_. Thoughts?

Reply:
Still, I think that `services.md` is a better fit. Missing support for pods is rather tech debt/implementation detail

Reply:
Yeah. If you look at `/v2/groups` it includes a group structure, and pods are in there. So I think the name service groups is more appropriate. @zen-dog maybe we'll need to highlight that pod dependencies / scaling doesn't work?

Reply:
sgtm, I've changed it. Btw, changes to the `index.md` can be made *at any time*, so let's not bike shed over a link that points to a void :)